io/disk/iozone.py

```diff
@@ -0,0 +1,649 @@
+import os
```

Comment:
Actually you can rather inspire here: https://github.com/avocado-framework/avocado-misc-tests/pull/26 for heading (both versions are acceptable)